packages/user-operation-controller/package.json

```diff
@@ -74,7 +74,7 @@
     "@metamask/gas-fee-controller": "^15.0.0",
     "@metamask/keyring-controller": "^16.0.0",
     "@metamask/network-controller": "^18.0.0",
-    "@metamask/transaction-controller": "^28.0.0"
+    "@metamask/transaction-controller": "^29.0.0"
```

Comment:
This is a breaking change, so we'll want to release this controller as well (otherwise it would be impossible to use the user-operation-controller without introducing a `peerDependencies` warning)